app/assets/stylesheets/login.scss

```diff
@@ -0,0 +1,62 @@
+@media (min-width: 768px) {
+  .omb_row-sm-offset-3 div:first-child[class*="col-"] {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector `omb_row-sm-offset-3` should be written in lowercase with hyphens